Feature: rename JELOS token to ROCKNIX

Sparticuz opened the pull request:
Rocknix replaced JELOS. Also added missing systems for it.

emmercm commented:
@Sparticuz thank you for this!

Here's what I'm thinking: it probably makes sense to add a new `{rocknix}` token for clarity, that for now will behave the same as `{jelos}` (same underlying config, as you have here). That would require updates to:

- Renaming `{jelos}` in docs/output/tokens.md
- Moving docs/usage/handheld/jelos.md to docs/usage/handheld/rocknix.md and adding a redirect in mkdocs.yml
- Renaming `{jelos}` in src/modules/argumentsParser.ts
- Adding `{rocknix}` to src/types/outputFactory.ts
- Adding `{rocknix}` to test/outputFactory.test.ts

the end result being a backward-compatible rename, where `{jelos}` can be phased out later, but for now will just be removed from documentation.

Is this something you'd like to try your hand at?

Sparticuz commented:
Feel free to take what I've done and build it out. I won't be able to contribute anytime soon

Sparticuz committed:
Feature: Add missing JELOS systems

Adds JELOS output tokens for VIC-20, Super Cassette Vision, J2ME,
Palm OS, PC-FX, TIC-80, CD-i, and PlayStation Vita.

emmercm committed:
Token rename, docs update
